Project-Bootcamp/router.js

```diff
@@ -0,0 +1,37 @@
+module.exports = ( function () {
+    var React = require('react');
+    var Router = require('react-router');
+    var Route = Router.Route;
+    var DefaultRoute = Router.DefaultRoute;
+
+    var MainView = require ('./views/main-view');
+    var HeaderView = require ('./views/header-view');
+    var RealmView = require ('./views/realm-view');
+    var CharacterView = require ('./views/character-view');
+    var ChallengerView = require ('./views/challenger-view');
+    var PvpView = require ('./views/pvp-view');
+
+    var ModuleRouter = function () {
+
+        this.routes = (
+            <Route handler={MainView}>
+                <DefaultRoute handler={HeaderView} />
```

Comment:
I'm not sure if you need to duplicate this on line 19, maybe you could just do:
``<DefaultRoute name="home" handler={HeaderView} />``